src/components/map-line/map-line.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Prop
+} from "@stencil/core";
+import { Component as GxComponent } from "../common/interfaces";
+import { polyline } from "leaflet/dist/leaflet-src.esm";
+import { parseCoords } from "../common/coordsValidate";
+
+@Component({
+  shadow: false,
+  //styleUrl: "map-line.scss",
```

Comment:
Let's remove this commented line as it doesn't add anything useful 😀

Reply:
Done. Addresed in 6aa1ac5